### PR TITLE
Alien whitelist update

### PR DESCRIPTION
**Pull request by radar651**
updates the alien whitelist to include Bicker98, CRG, Cyrstal, Vioras and Nerdass. Can be found on the virgo forms.

### CODE REVIEW

**Comment by Arokha** [comment]
Need links please.

**Comment by TheToaster98** [comment]
Bricker98
https://forum.vore-station.net/viewtopic.php?f=45&t=1753
CGR
https://forum.vore-station.net/viewtopic.php?f=45&t=1756
Crystal
https://forum.vore-station.net/viewtopic.php?f=45&t=1755
Vitoras
https://forum.vore-station.net/viewtopic.php?f=45&t=1751
Nerdass
https://forum.vore-station.net/viewtopic.php?f=45&t=1750

File: config/alienwhitelist.txt [hunk 1]
```diff
@@ -48,3 +48,9 @@ xioen - Diona
 xioen - Xenochimera
 zalvine - Shadekin Empathy
 zammyman215 - Vox
+Bricker98 - Protean
+CGR - Protean
+Storesund97 - Protean
+Vitoras - Protean
+Nerdass - Protean
```
Comment (TheFurryFeline) [inline config/alienwhitelist.txt:55]:
```suggestion
bricker98 - Protean
cgr - Protean
storesund97 - Protean
vitoras - Protean
nerdass - Protean
```
Like so

**Comment by KasparoVy** [comment]
So many speed demons, wow. KHI makin moves

**Comment by MrSign** [comment]
Hnggf, why wasn't this alphabetised like every other whitelist?

**Pull request merged**